qa-covid19.planx-pla.net/manifest.json

```diff
@@ -4,33 +4,33 @@
     "That's all I have to say"
   ],
   "versions": {
+    "ambassador": "quay.io/datawire/ambassador:1.4.2",
```

Comment:
is there a change to the versions, or only the order? i don't see the change if there's one

Reply:
Just the order. I've sorted them in alphabetical order.